src/multisig/multisig.cpp

```diff
@@ -80,8 +80,7 @@ namespace cryptonote
     crypto::secret_key blinded_skey = get_multisig_blinded_secret_key(keys.m_spend_secret_key);
     for (const auto &k: spend_keys)
     {
-      rct::key sk = rct::scalarmultKey(rct::pk2rct(k), rct::sk2rct(blinded_skey));
-      crypto::secret_key msk = get_multisig_blinded_secret_key(rct::rct2sk(sk));
+      crypto::secret_key msk = get_multisig_blinded_secret_key(rct::rct2sk(rct::scalarmultKey(rct::pk2rct(k), rct::sk2rct(blinded_skey))));
```

Comment:
The compiler is still freely allowed to use temporaries ... ?

Reply:
I'm not writing a compiler just for this though.

Reply:
My comment was about the change removing the temporary being just for show. The compiler isn't required to create a temporary address with the existing code, yet can still create one after this change. This is generally why these techniques are _nearly_ pointless, its hard to target every memory address in practice.

Reply:
OK. Then I guess it's best to make very simple statements, with explicit wipes. Agreed ?

Reply:
I've done that then.